### DISCUSSION TITLE
Add more playback speed options in the editor

### DISCUSSION THREAD

**Discussion by Enterpriseosu** [ppy/osu#13202] [Ideas]
**Describe the new feature:** 

Instead of having 25%/50%/75%/100% I think it's better to have more and espacially slower "stages" in the Editor.

**Proposal designs of the feature:** 

I think something like 10 "slow down stages" would be perfect for the editor: 10%/20%/.../90%/100%.

This would allow mappers to get a better and easier feeling on how to map certain stages in fast, high bpm songs.

**Comment by jai-x** [comment]
I think this might be a duplicate of https://github.com/ppy/osu/issues/2393 (unsure)

See also the reasoning given in this PR: https://github.com/ppy/osu/pull/12080

**Comment by Moltanicaa** [comment]
How reasonable is it to consider adding a speed of 125% (Double Time)?

**Comment by Enterpriseosu** [comment]
> How reasonable is it to consider adding a speed of 125% (Double Time)?

Not a bad idea but i think adding 110%/120%/.../150% will just end up in mappers concentrating even more on making DT Farm maps as some already do. My point was to have a slower playback speed to help mappers to emphasize specific parts in maps much better and easier.

**Comment by acikek** [comment]
> will just end up in mappers concentrating even more on making DT Farm maps as some already do

I don't think so... nonetheless, that's a bad justification to restrict a feature that could be helpful to mappers; I think 125% option is also a great idea along with the more granular playback speeds
